test/test_rack_server.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "test/unit"
+require "rack/test"
+require "xmlrpc/server"
+require 'xmlrpc/create'
+require 'xmlrpc/parser'
+
+class TestRack < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    s = XMLRPC::RackServer.new
+
+    s.add_handler("test.add") do |a,b|
+      a + b
+    end
+
+    s.add_handler("test.div") do |a,b|
+      if b == 0
+        raise XMLRPC::FaultException.new(1, "division by zero")
+      else
+        a / b
+      end
+    end
+
+    s.set_default_handler do |name, *args|
+      raise XMLRPC::FaultException.new(-99, "Method #{name} missing" +
+            " or wrong number of parameters!")
+    end
+
+    s.add_introspection
+
+    s
+  end
+
+  def test_successful_call
+    assert_equal([true, 9], call("test.add", 4, 5))
```

Comment:
Could you align arguments for easy to read?

```suggestion
    assert_equal([true, 9],
                 call("test.add", 4, 5))
```